ui/src/components/yaml/index.tsx

```diff
@@ -377,35 +380,45 @@ const Yaml = (props: IProps) => {
                   />
```

Comment:
Unnecessary nesting of div elements. Consider removing the extra parent div to simplify the structure.